config/envs/development.py

```diff
@@ -10,17 +10,4 @@
 class DevelopmentConfig(DefaultConfig):
 
     FSD_LOGGING_LEVEL = logging.DEBUG
-
-    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL")
```

Comment:
I think we want to keep this in to let you override the database locally?

Reply:
its in the default env so we shouldn't need to have it here too

Reply:
agreed 👍